librisworks/src/main/groovy/se/kb/libris/mergeworks/Doc.groovy

```diff
@@ -209,7 +209,10 @@ class Doc {
 
     boolean hasPart() {
         workData['hasPart'] || instanceData['hasTitle'].findAll { it['@type'] == 'Title' }.any {
-            it.hasPart?.size() > 1 || it.hasPart?.any { p -> asList(p.partName).size() > 1 || asList(p.partNumber).size() > 1 }
+            it.hasPart?.size() > 1
+                    || it.hasPart?.any { p -> asList(p.partName).size() > 1
+                    || asList(p.partNumber).size() > 1 }
+                    || [it.mainTitle, it.titleRemainder, it.subtitle].findAll().toString() =~ /(?<!\/.+)(\s+;)/
```

Comment:
Why `\/` in the regexp?

Reply:
We allow semicolon when it's preceded by a slash (not aggregate).